fault/verilator_target.py

```diff
@@ -730,6 +751,14 @@ def run(self, actions, verilator_includes=None, num_tests=0,
         else:
             env = None
 
+        # codegen the c++ binding if needed
+        if len(self.pysv_funcs) > 0:
+            header = os.path.join(self.directory, "pysv.hh")
+            pysv.generate_cxx_binding(self.pysv_funcs, filename=header)
+            lib_path = pysv.compile_lib(self.pysv_funcs, os.path.join(self.directory, "obj_dir"))
+            if env is None:
+                env = {"LD_LIBRARY_PATH": os.path.realpath(os.path.dirname(lib_path))}
```

Comment:
Should we have an else case here where we either insert LD_LIBRARY_PATH (if it doesn't exit) or append to it (if it exists)?